code/game/machinery/computer/tram_controls.dm

```diff
@@ -23,15 +23,13 @@
  * Locates tram parts in the lift global list after everything is done.
  */
 /obj/machinery/computer/tram_controls/proc/find_tram()
-	var/obj/structure/industrial_lift/tram/central/tram_loc = locate() in GLOB.lifts
-	tram_part = tram_loc //possibly setting to something null, that's fine, but
-	tram_part.find_our_location()
+	tram_part = GLOB.central_tram //possibly setting to something null, that's fine
```

Comment:
This should probably be a var on the computer instead of a global. That way we can have more than one tram on a map.

Reply:
I was thinking of making it like that but the original implementation was already somewhat similar to this, so I just found it easier to have only one global central tram.

If there is an actual need for more than one tram on a map, this can be implemented, but it really isn't needed right now.